index.html

```diff
@@ -215,6 +215,8 @@ <h4>LET'S CONNECT</h4>
         <div class="button">
           <button class="button1" type="button">Get my resume</button>
         </div>
+      </div>
+      <div class="third the-languages">
```

Comment:
- [ ] You are missing some images and icons in the about me section, kindly include them🙄👍
Your Implementation | Figma Design
:---:|:---:
![](https://user-images.githubusercontent.com/44624138/149247086-c374f851-181c-4b2a-8cea-191cd9d14661.png) | ![](https://user-images.githubusercontent.com/44624138/149247139-512dbcc4-e8b5-4ad6-a7f0-c115b3262e64.png)

Reply:
- [ ] As stated earlier Kindly make sure your buttons styling matches that of Figma😉.